build.gradle

```diff
@@ -50,20 +50,19 @@ dependencies {
     implementation 'software.amazon.awssdk:sqs'
     implementation 'software.amazon.awssdk:dynamodb'
 
-    // TODO: Transient dependency patches.
-    implementation 'ch.qos.logback:logback-core:1.2.13'
-    implementation 'ch.qos.logback:logback-classic:1.2.13'
-
     // TODO: Once we upgrade amazon-sqs-java-messaging-lib to v2 we can remove aws-java-sdk-core and sqs.
     implementation 'com.amazonaws:amazon-sqs-java-messaging-lib:1.1.2'
     implementation 'com.amazonaws:aws-java-sdk-core:1.12.368'
     implementation 'com.amazonaws:aws-java-sdk-sqs:1.12.368'
     implementation 'com.google.code.gson:gson:2.10.1'
-    implementation 'com.google.guava:guava:33.0.0-jre' // Explicit declaration to overide outdated/vulnerable dependency within WireMock
+    implementation 'com.google.guava:guava:33.0.0-jre' // Explicit declaration to override outdated/vulnerable dependency within WireMock
```

Comment:
Sorry, just for clarification - does bumping `wiremock` not resolve this or is this being used in an alternate dependency?